.circleci/soltest_all.sh

```diff
@@ -28,10 +28,46 @@ set -e
 
 REPODIR="$(realpath $(dirname $0)/..)"
 
-EVM=istanbul OPTIMIZE=1 ABI_ENCODER_V2=1 ${REPODIR}/.circleci/soltest.sh
+# This number has to equal the number of steps below.
```

Comment:
I bet that sooner or later this will no longer be the case without anyone noticing :-).

Reply:
It will actually make the test fail - see the end of the flie.

Reply:
Ah right, indeed!